_articles/importing/teams-enterprise-migration-guide.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: article
+title:  Teams and Enterprise Organization Migration Guide
+categories: [import-export]
+featured: false
+popular: false
+hidden: false
+tags: [import, teams, enterprise, onboarding, security]
+order: 98
```

Comment:
No need to bury this 🔥. We can set this to `order: 09`, and shift /import-faqs/ to `order: 10`.

Reply:
No worries, we can move around as needed 👍